README.md

```diff
@@ -1,7 +1,14 @@
-A bzl file that contains maven_jar and maven_aar rules for all artifacts in
-https://maven.google.com. Not guaranteed to be correct or up to date. Some of
-the artifacts depend on artifacts that are not present on
-https://maven.google.com. We ignore these and hope not to fail.
+# gmaven_rules
+
+This project generates `gmaven.bzl`, a file containing `maven_jar` and `maven_aar`
+rules for all artifacts in [https://maven.google.com](https://maven.google.com).
+
+# Support Policy
+
+This project is an **interim solution** during which Google Maven and AAR
```

Comment:
I think the phrasing of "solution during which" doesn't make sense. Suggestion:

This project is an interim solution that is supported until Google Maven and AAR support is added to bazel-deps

Reply:
done